seed/challenges/08-coding-interview-questions-and-take-home-assignments/coding-interview-data-structure-questions.json

```diff
@@ -2703,12 +2680,12 @@
       "title": "Directed and Undirected Graphs",
```

Comment:
@Greenheart changes made! The regex intro has only one slide. I split up my intro into like 6 slides...BUT it has some awesome graphics that will hopefully make it easier to understand graphs at the beginning.